support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepExecution.java

```diff
@@ -1,6 +1,7 @@
 package org.jenkinsci.plugins.workflow.support.steps;
 
 import com.google.inject.Inject;
+
```

Comment:
Please avoid gratuitous changes like this. A PR diff should be as short as possible while still accomplishing its stated goal. In particular, configure your IDE to not reformat otherwise unmodified lines of code. If your IDE has no option, you should manually revert gratuitous diff hunks by repeatedly running `git diff master` and restoring the original block, until your patch is minimized.